scenarios/08_Outlaw_Base.cfg

```diff
@@ -31,6 +31,8 @@
         canrecruit=yes
         side=1
         recruit=Fencer,Peasant,Bowman,Spearman
+        # Default gold when left unspecified is 100:
+        {GOLD 120 110 100}
```

Comment:
so, checking my replays, I finished on turns 19, 21, and 30 on each of my playthroughs (all on EASY), so, I guess since that last one was kind of close to the limit, I can raise it...
(current limit is `{TURNS 32 28 24}`, for reference, since that doesn't show in the previewed portion of this diff)

Reply:
changed to `{34 31 28}` in 4bdbac0